Source/Modules/Documentarian.MicrosoftDocs/Source/Public/Functions/ConvertTo-Contraction.ps1

```diff
@@ -0,0 +1,72 @@
+function ConvertTo-Contraction {
```

Comment:
All code files need the copyright and license headers.
```suggestion
# Copyright (c) Microsoft Corporation.
# Licensed under the MIT License.

function ConvertTo-Contraction {
```

Reply:
fixed